src/Components/WebAssembly/Build/src/targets/ServiceWorkerAssetsManifest.targets

```diff
@@ -178,7 +178,7 @@
 
   <Target Name="_OmitServiceWorkerContent"
     Condition="'$(ServiceWorkerAssetsManifest)' != ''"
-    BeforeTargets="AssignTargetPaths">
+    BeforeTargets="AssignTargetPaths;ResolveCurrentProjectStaticWebAssets">
```

Comment:
Why was this causing issues?

Reply:
It's subtle, but it fixes #20789!

Reply:
Ah I see. Thanks.

Reply:
#20789

It was not being taken out and it was causing a duplicated static web asset (which is not allowed)